modules/migration-editing-pvs-in-migplan.adoc

```diff
@@ -35,11 +50,15 @@ spec:
       name: mysql
       namespace: mysql-persistent
     selection:
-      action: copy
-      copyMethod: filesystem
-      storageClass: cephrbd <1>
-      accessMode: ReadWriteMany <2>
+      action: <copy> <1>
+      copyMethod: <filesystem> <2>
+      verify: true <3>
+      storageClass: <gp2> <4>
+      accessMode: <ReadWriteMany> <5>
     storageClass: cephfs
 ----
-<1> Destination storage class.
-<2> Destination access mode.
+<1> Allowed values are `move`, `copy`, and `skip`. If only one action is supported, the default value is the supported action. If multiple actions are supported, the default value is `copy`.
```

Comment:
```suggestion
<1> Allowed values are `move`, `copy`, and `skip`. If only one action is supported, the default value is the supported action. If multiple actions are supported, the default value is `copy`.
```
Allowed or Accepted?  Maybe change to accepted or supported values? 

I know per the IBM style guide, we shouldn't use the word "allow" but I'm unsure about "allowed values". 

> Avoid` stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

Reply:
"Allowed values" occurs 21 times in OCP docs. "Accepted values" occurs 3 times.

Reply:
The IBM style guide quote does not apply because we are using "allowed" as an adjective, not a verb. "Permitted values" would also work but it is never used in OCP docs.